multistream.js

```diff
@@ -0,0 +1,98 @@
+'use strict'
```

Comment:
This file is a duplicate of `benchmarks/multistream.js`?

Reply:
removed, sorry about it